pkg/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	registry            = prometheus.NewRegistry()
+	storageReconfigured = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "image_registry_operator",
+		Subsystem: "storage",
+		Name:      "reconfigured",
+		Help:      "Number of times registry storage got reconfigured.",
+	})
+	degraded = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "image_registry_operator",
+		Subsystem: "status",
+		Name:      "degraded",
```

Comment:
```suggestion
		Name:      "degraded_total",
```

Reply:
I can't accept this proposal here as it will break things. I need to update the tests and also the query done using this parameter inside manifests. I am going to send a commit with this update.

Reply:
Done.